src/platform/wii/main.c

```diff
@@ -113,6 +113,7 @@ static uint32_t referenceRetraceCount;
 static bool frameLimiter = true;
 static int scaleFactor;
 static unsigned corew, coreh;
+static bool m_reset;
```

Comment:
Why `m_`? This isn't a private member of a C++ class. Also, this needs to be initialized to false.